src/components/Table/TasksTable.tsx

```diff
@@ -143,14 +143,11 @@ const getColumnDefsControlled = (
   }));
 };
 
-const renderStatusBadge = (
-  status: string,
-  { blocked }: Task
-): null | JSX.Element => {
+const renderStatusBadge = (status: string): null | JSX.Element => {
   if (status === "" || !status) {
```

Comment:
we could probably add this logic to the TaskStatus badge and replace renderStatusBadge usage with TaskStatusBadge